create_portal_teaching_dataset.py

```diff
@@ -1,3 +1,10 @@
+# -*- coding: utf-8 -*-
+"""
+Updated on Mon Mar  5 21:56:52 2018 by Katrin Tirok
+Corrected one wrong date in surveys table
+Corrected url for download of species and plot tables
+"""
+
```

Comment:
This information belongs in the commit message not the file. Unless there's a reason to specific to provide the encoding it shouldn't be included and typically `utf-8` is the default. In short, I think we can just remove this seven line change.